x-pack/test/security_solution_api_integration/scripts/mki_api_ftr_execution.ts

```diff
@@ -56,6 +64,25 @@ function executeCommand(command: string, envVars: any, workDir: string): Promise
   });
 }
 
+async function parseProductTypes(log: ToolingLog): Promise<ProductType[] | undefined> {
+  if (!process.env.TARGET_SCRIPT) {
+    log.error('TARGET_SCRIPT environment variable is not provided. Aborting...');
+    return process.exit(1);
+  }
+
+  const apiConfigs = JSON.parse(fs.readFileSync('./scripts/api_configs.json', 'utf8'));
```

Comment:
Should we consider using the asynchronous version of file reading?

Reply:
since we're inside an async function I think we should